python_modules/dagster/dagster/core/definitions/repository_definition.py

```diff
@@ -788,6 +789,11 @@ def from_list(
             else:
                 pipelines[name] = pipeline_or_job
 
+        if default_executor_def:
+            for name, job_def in jobs.items():
+                if job_def.executor_def == multi_or_in_process_executor:
```

Comment:
What if someone sets a default executor def and then also sets `multi_or_in_process_executor` on one of their jobs?

Reply:
In this world, that executor would get overridden. I guess we could protect against that?